examples/install/cluster-operator/04-ClusterRoleBinding-strimzi-cluster-operator-topic-operator-delegation.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
+metadata:
+  name: strimzi-cluster-operator-topic-operator-delegation
```

Comment:
All rights from `strimzi-topic-operator` seem to be already in `strimzi-cluster-operator`. Do we really need this binding when we already have all these rights?

Reply:
Functionally, no we don't. But I think it's much clearer if we keep separate the permissions the CO itself needs from the ones it needs only so it can delegate them to other component. For example, with the `Topic` CRD arriving soon the CO will need permissions on `Topics`, but only so it can bind them to the TO's SA, rather than because it will manipulate `Topic`s itself.

Reply:
Ok, fine with me. I can live with it.